unitTests/ui/MissingTermsTest.ts

```diff
@@ -132,9 +133,9 @@ export function MissingTermsTest() {
           it will not be displayed as a missing term.`, () => {
             const wordBoundarysCharacter = getWordBoudaryChars();
             wordBoundarysCharacter.forEach(character => {
-              const boundaryCharacter = `\\${character}s`;
-              const query = `efile${boundaryCharacter} `;
-              fakeResult.absentTerms = [`${boundaryCharacter}`];
+              const boundaryCharacter = `${character}s`;
+              const query = `efile\\${boundaryCharacter} `;
+              fakeResult.absentTerms = [`\\${boundaryCharacter}`];
```

Comment:
I struggled with this test and the one on line 278, in particular because the description says `it will not be displayed as a missing term.`, yet it is expecting an array populated with truthy values

Since pre-change we were returning the values in the `absentTerms` array we pass in (line 138), passing an escaped back-slash made it's way into the final `missingTerms` array. Post-change, the value we return is what the regex matches, which an unescaped slash.